packages/component-library/components/Button/index.ts

```diff
@@ -1,2 +1,3 @@
 export { default as Button } from "./Button"
 export { default as IconButton } from "./IconButton"
+export { ButtonProps } from "./components/GenericButton"
```

Comment:
Would love for all of our props to be exported in a similar way. No changes required from you, but just a thought for a future pattern.